Makefile

```diff
@@ -34,20 +34,24 @@ RUN = $(DOCKER) run \
   --rm \
   -i $(ENABLE_TTY) `$(MAKE_DOCKER)`
 
+BUILD_TYPE ?= Debug
```

Comment:
This is already done at https://github.pie.apple.com/Home-ADK/ADK/blob/master/Build/Makefile#L10 so unnecessary here.

Reply:
I removed it.